lib/broccoli/angular2-app.js

```diff
@@ -55,7 +56,13 @@ Angular2App.prototype.toTree = function () {
     //     outputFile: '/app.js'
     // });
 
-    return mergeTrees([assetTree, tsSrcTree, tsTree, jsTree, this.index(), vendorJsTree], { overwrite: true });
+    var lessTree = new Funnel(sourceTree, {
+        include: ['**/*.less'],
+        allowEmpty: true
+    });
+    var lessSrcTree = compileLess(lessTree);
+
+    return mergeTrees([assetTree, tsSrcTree, tsTree, jsTree, this.index(), lessSrcTree, vendorJsTree], { overwrite: true });
```

Comment:
The tricky thing about here is that we would need to pull the config from somewhere - the only approaches I see right now is to check for the existence of a less file in a component directory (a hack), or save the config somewhere when generating the component. Saving the config somewhere might be ok, since we should be able to safely assume that a user is going to be only using one CSS preprocessor ever, but then comes the question of how should we be saving it. Should we save that information right in the generated package.json?